gateways/clients/wallet_service_db_client.py

```diff
@@ -105,10 +110,19 @@ def get_address_history(self, address: str, token: str, limit: int, offset: int)
 
         return result
 
-    def get_address_tokens(self, address: str, limit: int, offset: int) -> List[dict]:
+    def get_address_tokens(self, address: str, limit: int, offset: int) -> Tuple[int, List[dict]]:
```

Comment:
Could you add a short description of what this function does?

I think it could make reading the code easier for someone that isn't used to it.

Reply:
Added docstrings explaining the method logic and some comments

Reply:
Thanks